pkg/util/installer/certs.go

```diff
@@ -0,0 +1,76 @@
+package installer
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+
+	utilpem "github.com/Azure/ARO-RP/pkg/util/pem"
+)
+
+// CertInterface contains cert.
+// See openshift/installer/pkg/asset/tls/tls.go
+type CertInterface interface {
+	// Cert returns the certificate.
+	Cert() []byte
+}
+
+// CertKeyInterface contains a private key and the associated cert.
+// See openshift/installer/pkg/asset/tls/tls.go
+type CertKeyInterface interface {
+	CertInterface
+	// Key returns the private key.
+	Key() []byte
+}
```

Comment:
do these need to be interfaces? or is it just so we can use the type from installer directly?